code/modules/reqs/supplypacks.dm

```diff
@@ -815,7 +815,7 @@ MEDICAL
 	access = ACCESS_MARINE_MEDBAY
 
 /datum/supply_packs/medical/medvac
-	name = "medvac system"
+	name = "medEvac system"
```

Comment:
Technically, it would be MEDEVAC with all caps.

Reply:
at least it's now have letter E in the name, but medevac in all caps sounds like unga, so yea, let's change it